specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/EasyAuthConfigs.json

```diff
@@ -0,0 +1,706 @@
+          "type": "string"
+        },
+        "clientSecretCertificateIssuer": {
+          "description": "An alternative to the client secret thumbprint, that is the issuer of a certificate used for signing purposes. This property acts as\na replacement for the Client Secret Certificate Thumbprint. It is also optional.",
+          "type": "string"
+        }
+      }
+    },
+    "AzureActiveDirectoryLogin": {
+      "description": "The configuration settings of the Azure Active Directory login flow.",
+      "type": "object",
+      "properties": {
+        "loginParameters": {
+          "description": "Login parameters to send to the OpenID Connect authorization endpoint when\na user logs in. Each parameter must be in the form \"key=value\".",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "disableWwwAuthenticate": {
+          "description": "<code>true</code> if the www-authenticate provider should be omitted from the request; otherwise, <code>false</code>.",
+          "type": "boolean"
+        }
+      }
+    },
+    "AzureActiveDirectoryValidation": {
+      "description": "The configuration settings of the Azure Active Directory token validation flow.",
+      "type": "object",
+      "properties": {
+        "allowedAudiences": {
+          "description": "The list of audiences that can make successful authentication/authorization requests.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "Facebook": {
+      "description": "The configuration settings of the Facebook provider.",
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "description": "<code>false</code> if the Facebook provider should not be enabled despite the set registration; otherwise, <code>true</code>.",
+          "type": "boolean"
+        },
+        "registration": {
+          "$ref": "#/definitions/AppRegistration",
+          "description": "The configuration settings of the app registration for the Facebook provider."
+        },
+        "graphApiVersion": {
+          "description": "The version of the Facebook api to be used while logging in.",
+          "type": "string"
+        },
+        "login": {
+          "$ref": "#/definitions/LoginScopes",
+          "description": "The configuration settings of the login flow."
+        }
+      }
+    },
+    "AppRegistration": {
+      "description": "The configuration settings of the app registration for providers that have app ids and app secrets",
+      "type": "object",
+      "properties": {
+        "appId": {
+          "description": "The App ID of the app used for login.",
+          "type": "string"
+        },
+        "appSecretRefName": {
```

Comment:
>appSecretRefName [](http://example.com/codeflow?start=9&length=16)

If this is a secret value it should have the x-ms-secret extension, and is not allowed in GET responses. If a secret need to be retrieved, it should be via a dedicated POST action.

Reply:
It is not secret value, it is just key(ref) name of secret which could be retrieved by another API